std/windows/syserror.d

```diff
@@ -135,16 +133,41 @@ class WindowsException : Exception
                 buf.put(": ");
         }
 
-        if (code)
+        if (code && writeErrorMessage(code, buf))
         {
-            auto success = putSysError(code, buf);
-            formattedWrite(buf, success ? " (error %d)" : "Error %d", code);
+            buf.put(" (error ");
+            toTextRange(code, buf);
+            buf.put(')');
         }
 
-        super(buf.data, file, line);
+        super(cast(immutable) buf.data, file, line);
     }
 }
 
+/// Writes the error string associated to `code` into `buf`.
+/// Writes `Error <code>` when the error message lookup fails
+private bool writeErrorMessage(DWORD code, ref Appender!(char[]) buf, int langId = 0) nothrow
```

Comment:
D style: avoid default arguments

Reply:
[D style](https://dlang.org/dstyle.html) doesn't mention default arguments?

Reply:
I was confused with the [DMD best practices](https://github.com/dlang/dmd/blob/master/CONTRIBUTING.md#dmd-best-practices). Still, I don't think a default argument is useful here.

Reply:
Feel free to ignore though, it's a nitpick.

Reply:
I think in this case it's because the original function has default arguments:

https://github.com/dlang/phobos/blob/6e913a4d7c0ad3fc9e863acb12f22ced8b7db347/std/windows/syserror.d#L91

But for a private function maybe removing the argument is better if it's not used within the same module.

Reply:
Removed the default argument, it isn't necessary ATM